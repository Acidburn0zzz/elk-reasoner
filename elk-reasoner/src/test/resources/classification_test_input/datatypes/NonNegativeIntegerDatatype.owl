Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/Test.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)

Ontology(<http://www.semanticweb.org/Test.owl>

EquivalentClasses(DataHasValue(:dp "0"^^xsd:nonNegativeInteger)		:_Zero1)
EquivalentClasses(DataHasValue(:dp "+0"^^xsd:nonNegativeInteger)	:_Zero2)

EquivalentClasses(DataHasValue(:dp "10"^^xsd:nonNegativeInteger)	:_Ten1)
EquivalentClasses(DataHasValue(:dp "+10"^^xsd:nonNegativeInteger)	:_Ten2)

EquivalentClasses(:NNInteger_Larger_Or_Equal_Than_10 	DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minInclusive "10"^^xsd:nonNegativeInteger)))
EquivalentClasses(:NNInteger_Larger_Than_10 		DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minExclusive "10"^^xsd:nonNegativeInteger)))
EquivalentClasses(:NNInteger_Less_Or_Equal_Than10 	DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:maxInclusive "10"^^xsd:nonNegativeInteger)))
EquivalentClasses(:NNInteger_LessThan10 		DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:maxExclusive "10"^^xsd:nonNegativeInteger)))

EquivalentClasses(:NNIntegers_Between_0_and_10_Ex 	DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minExclusive "0"^^xsd:nonNegativeInteger xsd:maxExclusive "10"^^xsd:nonNegativeInteger)))
EquivalentClasses(:NNIntegers_Between_0_and_10_Inc 	DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minInclusive "0"^^xsd:nonNegativeInteger xsd:maxInclusive "10"^^xsd:nonNegativeInteger)))
EquivalentClasses(:NNIntegers_Between_0_and_10_IncEx 	DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minInclusive "0"^^xsd:nonNegativeInteger xsd:maxExclusive "+10"^^xsd:nonNegativeInteger)))
EquivalentClasses(:NNIntegers_Between_0_and_10_ExInc 	DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minExclusive "0"^^xsd:nonNegativeInteger xsd:maxInclusive "+10"^^xsd:nonNegativeInteger)))

EquivalentClasses(:NNIntegers_Between_1_and_9_Ex 	DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minExclusive "1"^^xsd:nonNegativeInteger xsd:maxExclusive "9"^^xsd:nonNegativeInteger)))
EquivalentClasses(:NNIntegers_Between_1_and_9_Inc 	DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minInclusive "1"^^xsd:nonNegativeInteger xsd:maxInclusive "9"^^xsd:nonNegativeInteger)))
EquivalentClasses(:NNIntegers_Between_1_and_9_IncEx 	DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minInclusive "1"^^xsd:nonNegativeInteger xsd:maxExclusive "+9"^^xsd:nonNegativeInteger)))
EquivalentClasses(:NNIntegers_Between_1_and_9_ExInc 	DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minExclusive "1"^^xsd:nonNegativeInteger xsd:maxInclusive "+9"^^xsd:nonNegativeInteger)))

EquivalentClasses(:InconsistentNNInteger1		DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:minExclusive "5"^^xsd:nonNegativeInteger xsd:maxExclusive "0"^^xsd:nonNegativeInteger)))
EquivalentClasses(:InconsistentNNInteger2		DataSomeValuesFrom(:dp DatatypeRestriction(xsd:nonNegativeInteger xsd:maxExclusive "0"^^xsd:nonNegativeInteger)))
)