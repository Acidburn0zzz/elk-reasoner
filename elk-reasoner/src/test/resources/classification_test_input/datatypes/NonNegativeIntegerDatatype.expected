Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/Test.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)

Ontology(<http://www.semanticweb.org/Test.owl>

EquivalentClasses(:NNInteger_LessThan10			:NNIntegers_Between_0_and_10_IncEx)
EquivalentClasses(:NNInteger_Less_Or_Equal_Than10	:NNIntegers_Between_0_and_10_Inc)
EquivalentClasses(:NNIntegers_Between_0_and_10_Ex	:NNIntegers_Between_1_and_9_Inc)
EquivalentClasses(:NNIntegers_Between_0_and_10_Inc	:NNInteger_Less_Or_Equal_Than10)
EquivalentClasses(:NNIntegers_Between_0_and_10_IncEx	:NNInteger_LessThan10)
EquivalentClasses(:NNIntegers_Between_1_and_9_Inc	:NNIntegers_Between_0_and_10_Ex)

SubClassOf(:NNInteger_Larger_Than_10			:NNInteger_Larger_Or_Equal_Than_10)
SubClassOf(:NNInteger_LessThan10			:NNInteger_Less_Or_Equal_Than10)
SubClassOf(:NNInteger_LessThan10			:NNIntegers_Between_0_and_10_Inc)

SubClassOf(:NNIntegers_Between_0_and_10_Ex		:NNInteger_LessThan10)
SubClassOf(:NNIntegers_Between_0_and_10_Ex		:NNIntegers_Between_0_and_10_ExInc)
SubClassOf(:NNIntegers_Between_0_and_10_Ex		:NNIntegers_Between_0_and_10_IncEx)
SubClassOf(:NNIntegers_Between_0_and_10_ExInc		:NNInteger_Less_Or_Equal_Than10)
SubClassOf(:NNIntegers_Between_0_and_10_ExInc		:NNIntegers_Between_0_and_10_Inc)
SubClassOf(:NNIntegers_Between_0_and_10_IncEx		:NNInteger_Less_Or_Equal_Than10)
SubClassOf(:NNIntegers_Between_0_and_10_IncEx		:NNIntegers_Between_0_and_10_Inc)

SubClassOf(:NNIntegers_Between_1_and_9_Ex		:NNIntegers_Between_1_and_9_ExInc)
SubClassOf(:NNIntegers_Between_1_and_9_Ex		:NNIntegers_Between_1_and_9_IncEx)
SubClassOf(:NNIntegers_Between_1_and_9_ExInc		:NNIntegers_Between_0_and_10_Ex)
SubClassOf(:NNIntegers_Between_1_and_9_ExInc		:NNIntegers_Between_1_and_9_Inc)
SubClassOf(:NNIntegers_Between_1_and_9_Inc		:NNInteger_LessThan10)
SubClassOf(:NNIntegers_Between_1_and_9_Inc		:NNIntegers_Between_0_and_10_ExInc)
SubClassOf(:NNIntegers_Between_1_and_9_Inc		:NNIntegers_Between_0_and_10_IncEx)
SubClassOf(:NNIntegers_Between_1_and_9_IncEx		:NNIntegers_Between_0_and_10_Ex)
SubClassOf(:NNIntegers_Between_1_and_9_IncEx		:NNIntegers_Between_1_and_9_Inc)

EquivalentClasses(:_Ten1 :_Ten2)
SubClassOf(:_Ten1	:NNInteger_Larger_Or_Equal_Than_10)
SubClassOf(:_Ten1	:NNIntegers_Between_0_and_10_ExInc)
SubClassOf(:_Ten2	:NNInteger_Larger_Or_Equal_Than_10)
SubClassOf(:_Ten2	:NNIntegers_Between_0_and_10_ExInc)

EquivalentClasses(:_Zero1 :_Zero2)
SubClassOf(:_Zero1	:NNInteger_LessThan10)
SubClassOf(:_Zero1	:NNIntegers_Between_0_and_10_IncEx)
SubClassOf(:_Zero2	:NNInteger_LessThan10)
SubClassOf(:_Zero2	:NNIntegers_Between_0_and_10_IncEx)

EquivalentClasses(:InconsistentNNInteger1 :InconsistentNNInteger2 owl:Nothing)
)
